examples/promise-co-await/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["es2015", "stage-2"],
```

Comment:
Absolutely.

You need to be using:

* [`babel-plugin-transform-async-to-generator`](http://babeljs.io/docs/plugins/transform-async-to-generator/)

Unless I am missing something, thats it. Which Node.js versions are you targeting?